### PR TITLE
update LICENSE

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Resolves https://github.com/keptn/keptn/issues/2725

Signed-off-by: jetzlstorfer <juergen.etzlstorfer@dynatrace.com>

### CODE REVIEW

**Review by warber — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Google Inc. All Rights Reserved.
+   Copyright 2016 Google Inc. All Rights Reserved. Content by The Keptn Authors
```
Comment (christian-kreuzberger-dtx) [inline LICENSE:190]: This is weird... Why do we attribute Google a copyright of this repository? Is it best practice that we have Google Code Labs in our repo, and the docs are kind of a 2nd class citizen? Should we restructure the repository?

Comment (johannes-b) [reply LICENSE:190]: Based on a quick search I found: https://softwareengineering.stackexchange.com/questions/304874/declaring-multiple-licences-in-a-github-project

Comment (jetzlstorfer) [reply LICENSE:190]: Thanks, I'll take a look and do some changes accordingly

**Comment by jetzlstorfer** [comment]
I've updated it. Please take another look @christian-kreuzberger-dtx

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**